packages/runtime/test/consumption/openid4vc.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
@Magnus-Kuhn we made this change on purpose to not have the whole file excluded of this rule.